src/toil/jobStores/abstractJobStore.py

```diff
@@ -270,19 +275,19 @@ def _exportFile(self, otherCls, jobStoreFileID, url):
         with self.readFileStream(jobStoreFileID) as readable:
             otherCls._writeToUrl(readable, url)
 
-    @abstractclassmethod
+    @classmethod
```

Comment:
Let's leave this abstract and push your implementation down into a JobStoreSupport class in the same module as the AbstractJobStore.

```
class JobStoreSupport(AbstractJobStore):
…
```